aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,178 @@
 
-.. image:: pubIIS/_static/rs.png
+  On the application's physical path, there are sensitive files that one would never want to accidently serve, such as **my_application.runtimeconfig.json**, **my_application.xml** (XML Documentation comments), and **my_application.deps.json**. Without the `web.config` file present to create the reverse proxy to Kestrel, IIS would serve these sensitive files openly on the Internet. **Therefore, it is important that the web.config file is never accidently renamed or removed from the deployment.**
 
-2. On the **Role Services** step, remove any items you don't need. The defaults are shown below.
+  In order to mitigate the risk of serving sensitive application files if `web.config` were ever accidently renamed or removed, one may add sensitive files to the **Hidden Segments** of **Request Filtering** or add blanket file extension rules to the **File Name Extensions** of **Request Filtering**. See `Hidden Segments \<hidden Segments\> <https://www.iis.net/configreference/system.webserver/security/requestfiltering/hiddensegments>`__ and `File Name Extensions \<fileExtensions\> <https://www.iis.net/configreference/system.webserver/security/requestfiltering/fileextensions>`__ for more information.
 
-.. image:: pubIIS/_static/role-services.png
+  Keep in mind that if you use the **File Name Extensions** approach that your application will not receive requests nor be able to serve any file with that extension, even static files in your **webroot** that you wish to serve (e.g., JSON or XML files you have placed in `wwwroot` for legitimate file serving). Therefore, specifically naming sensitive files using the **Hidden Segments** approach might be preferable in most scenarios [i.e., explicitly naming **my_application.runtimeconfig.json**, **my_application.xml** (if any XML Documentation comments files are present), **my_application.deps.json**, and any other files that are not explicitly excluded by IIS for static file serving].
 
-.. _unlock-handlers:
+Configure the Website in IIS
+----------------------------
 
-3. Unlock the configuration section.
+#. In **IIS Manager**, create a new website. Provide a **Site name** and set the **Physical path** to the application's assets folder that you created. Provide the **Binding** configuration and create the website.
+#. Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
 
-  - Launch IIS Manager and select the server in the **Connections** pane on the left (see image below).
-  - Double-click **Configuration Editor**.
-  - In the **Section** drop-down, select **system.webServer/handlers**, and then click **Unlock Section**.
+.. note:: If you change the default identity of the application pool from **ApplicationPoolIdentity** to some other user account, make sure you configure the correct permissions for the user identity to use the application's assets folder.
```

Comment:
drop "to some other user account"
consider
If you change the default identity of the application pool from **ApplicationPoolIdentity**, verify the new identity has the required permissions to access the application assets an database.